resources/testdata/WghtVar-Bold.ufo/features.fea

```diff
@@ -0,0 +1,10 @@
+languagesystem DFLT dflt;
+
+feature ss01 {
+        featureNames {
+        name "my fun feature";
+    };
+
+    sub bar by plus;
+} ss01;
+
```

Comment:
Does fea_rs support [`table name {} name`](https://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html#9.e) blocks? If so, I think it deserves a test, as they would override existing names in the font.

Reply:
Names defined in the feature file override those already in the name table. This [what feaLib does](https://github.com/fonttools/fonttools/blob/b90ac3c29f6030ec7309b044c41a72d9f586163f/Lib/fontTools/feaLib/builder.py#L452), and I believe this is what makeotf does as well (though it does not allow overriding name IDs 2 and 6, or at least that [what the spec says](https://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html#:~:text=are%20reserved%20by%20the%20implementation%20and%20cannot%20be%20overridden), I haven’t tested it recently).